boefjes/tests/conftest.py

```diff
@@ -111,16 +113,35 @@ def _task_from_id(self, task_id: UUID):
         return self._popped_items[str(task_id)]
 
     def push_item(self, p_item: Task) -> None:
-        self._pushed_items[str(p_item.id)] = p_item
+        self._pushed_items[str(p_item.id)] = [p_item]
+
+
+class MockBytesAPIClient:
+    def __init__(self):
+        self.queue = Manager().Queue()
+
+    def save_boefje_meta(self, boefje_meta: BoefjeMeta) -> None:
+        self.queue.put(("save_boefje_meta", (boefje_meta.model_dump(),)))
+
+    def save_raw(self, boefje_meta_id: str, raw: str | bytes, mime_types: set[str] = frozenset()) -> UUID:
```

Comment:
I think the type `set[str]` is incorrect, since a `frozenset` is not a subclass of `set`